<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutClient</class>
 <widget class="QDialog" name="ajoutClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="b_valider">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_annuler">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>451</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Identité</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Localisation</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>400</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="l_adresse">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="q_adresse"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_ville">
       <property name="text">
        <string>Ville</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="q_ville"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="l_codePostal">
       <property name="text">
        <string>Code Postal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="q_codePostal">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>451</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Contacts</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="l_courriel">
       <property name="text">
        <string>Courriel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="q_courriel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="l_telephone">
       <property name="text">
        <string>Téléphone</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="q_telephone"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="l_typeAnnonce">
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="genre">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <item>
       <property name="text">
        <string>Mr.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mlle.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mme.</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="l_nom">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="q_nom">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="l_prenom">
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="q_prenom">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="l_adresse_2">
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="date"/>
    </item>
   </layout>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>l_typeAnnonce</zorder>
  <zorder>genre</zorder>
  <zorder>b_valider</zorder>
  <zorder>b_annuler</zorder>
  <zorder>q_nom</zorder>
  <zorder>q_prenom</zorder>
  <zorder>date</zorder>
  <zorder>l_adresse_2</zorder>
  <zorder>l_nom</zorder>
  <zorder>l_prenom</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
